docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -0,0 +1,119 @@
+
+Dockstore tools are more associated with creating/owning Docker images that are used in conjunction with a descriptor language, and Dockstore workflows are more closely tied to the descriptor files themselves. If you control the Docker image being used by your CWL program, and want verisoning to be based upon that images tags, you should probably write a tool. If you do not control the Docker image being used, and/or are not writing in CWL, you should probably write a workflow. Of course, everyone's use case is different. We encourage you to look around Dockstore for inspiration for your next bioinformatics project.
+
+Overview
+--------
+
++------------------------+------------------------------------------+-------------------------------------------------+
+| Language               | Tool                                     | Workflow                                        |
++========================+==========================================+=================================================+
+| CWL                    |  - Class: CommandLineTool                |  - Class: Workflow                              |
+|                        |  - Dockerfile for the tool's image       |  - No Dockerfile required                       |
++------------------------+------------------------------------------+-------------------------------------------------+
+| WDL                    |  - One task that runs in a Docker image  |  - ≥1 task, which could run in Docker image     |
+|                        |  - A workflow section that runs the task |  - A workflow section that runs the task(s)     |
+|                        |  - The Dockerfile for the task's image   |  - No Dockerfile required                       |
++------------------------+------------------------------------------+-------------------------------------------------+
+| Nextflow               | N/A                                      | Any valid Nextflow workflow                     |
++------------------------+------------------------------------------+-------------------------------------------------+
+| Galaxy                 | N/A*                                     | Any valid Galaxy workflow                       |
++------------------------+------------------------------------------+-------------------------------------------------+
+
+\* There are tools that make up Galaxy workflows from the Galaxy toolbox or ToolShed.
+Dockstore does not support registration of these tools.
+
+
++------------------------+------------------------------------------+-------------------------------------------------+
+| Dockstore Support      | Tool                                     | Workflow                                        |
++========================+==========================================+=================================================+
+| Desciptor File         |  Just one file (ie, one .wdl file)       |  Can be multiple files (ie, ten .wdl files)     |
```

Comment:
Not exactly right, think @coverbeck  is going to link this but refer to 
https://app.slack.com/archives/C16ET3CF4/p1643404192580239